app/services/course/assessment/question/programming/cpp/cpp_package_service.rb

```diff
@@ -0,0 +1,248 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::Programming::Cpp::CppPackageService < \
```

Comment:
I understand you had an offline conversation with @allenwq yesterday. Maybe you can work on that first, and keep this at the back of your head? Pushing first and then refactoring later seems ok, unless it makes testing for you too painful, then maybe splitting it up might be easier.